src/Template/Provider/BaseProvider.php

```diff
@@ -14,7 +14,7 @@
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
```

Comment:
Can be reverted.

Reply:
Hm. I realise that you have these excessive spaces all over. But this topic seems out of scope. Reverting as requested.

Reply:
Yeah, that's probably a bug in my live template :D Let's address this in another change.